packages/gp2-components/src/organisms/WorkingGroupCard.tsx

```diff
@@ -20,11 +21,11 @@ const { getCounterString } = utils;
 
 type WorkingGroupCardProps = Pick<
   gp2Model.WorkingGroupResponse,
-  'id' | 'title' | 'members' | 'shortDescription' | 'leadingMembers'
+  'id' | 'title' | 'members' | 'shortDescription' | 'leadingMembers' | 'tags'
 >;
 const containerStyles = css({
   display: 'flex',
-  gap: rem(16),
+  gap: rem(24),
```

Comment:
I think we can't change all to 24, because some gaps like this one in purple is 16px
<img width="789" alt="Screenshot 2024-03-25 at 14 15 03" src="https://github.com/yldio/asap-hub/assets/16595804/8244d16a-e8f7-4c65-89b4-1ad68c81afbf">